CONTRIBUTORS

```diff
@@ -1,4 +1,4 @@
-======================================
+update_workers======================================
```

Comment:
I believe this line was changed by mistake. Wasn't it?

```suggestion
======================================
```

Reply:
For some reason, GitHub renders this suggestion incorrectly.

Reply:
Yes, `update_workers` should be removed